cmd/antctl/main.go

```diff
@@ -26,6 +26,7 @@ import (
 	"k8s.io/component-base/logs"
 
 	"antrea.io/antrea/pkg/antctl"
+	_ "antrea.io/antrea/pkg/antctl/raw/multicluster/get"
```

Comment:
Instead of leveraging the obscure side effect of importing and adding the package of one subcommand here while other subcommands are initialized in another fashion, it would be better to make the `multicluster.GetCmd` initialized in its own package and the initialization should be done automatically when importing the command. Otherwise it would lead to an unmanageable maintenance effort to its consumer: "If you want to use A, you should also import B, C, etc.

I would suggest to move `init` function to `pkg/antctl/raw/multicluster/commands.go`:
```
var GetCmd = &cobra.Command{
	Use:   "get",
	Short: "Display one or many resources in a ClusterSet",
}

func init() {
	GetCmd.AddCommand(get.NewClusterSetCommand())
	GetCmd.AddCommand(get.NewResourceExportCommand())
	GetCmd.AddCommand(get.NewResourceImportCommand())
}
```